docs/application/native/guides/multimedia/face-recognition.md

```diff
@@ -0,0 +1,172 @@
+   if (ret != IMAGE_UTIL_ERROR_NONE) {
+       // handle an error.
+   }
+
+   ret = image_util_decode_set_colorspace(imageDecoder, IMAGE_UTIL_COLORSPACE_RGB888);
+   if (ret != IMAGE_UTIL_ERROR_NONE) {
+       // handle an error.
+   }
+
+   /* Decode image and fill the image data to mv_source handle */
+   snprintf(filePath, 1024, "/path/to/face_image.jpg");
+   ret = image_util_decode_set_input_path(imageDecoder, filePath);
+   if (ret != IMAGE_UTIL_ERROR_NONE) {
+       // handle an error.
+   }
+
+   ret = image_util_decode_set_output_buffer(imageDecoder, &dataBuffer);
+   if (ret != IMAGE_UTIL_ERROR_NONE) {
+       // handle an error.
+   }
+
+   ret = image_util_decode_run(imageDecoder, &width, &height, &bufferSize);
+   if (ret != IMAGE_UTIL_ERROR_NONE) {
+       // handle an error.
+   }
+
+   ret = mv_source_fill_by_buffer(mv_source, dataBuffer, (unsigned int)bufferSize,
+                                 (unsigned int)width, (unsigned int)height, MEDIA_VISION_COLORSPACE_RGB888);
+   if (ret != MEDIA_VISION_ERROR_NONE) {
+      free(dataBuffer);
+	  // handle an error.
+   }
+
+   ret = image_util_decode_destroy(imageDecoder);
+   if (ret != IMAGE_UTIL_ERROR_NONE) {
+       // handle an error.
+   }
+
+   free(dataBuffer);
+   ```
+
+3. Now, we are ready for using the face recognition main features. There are three face recognition features - register, unregister and recognize.
+   You can use one of them.
+ 
+   3.1 Register a new face, which is a process that the face recognition framework tains and generates
+       a internal model file with the given face data and its label. You could repeat this step to train
+	   the internal model file with more faces. For the accuracy, we recommend to train at least three images per a face.
+
+   ```
+   // Train the internal model with a given image - mv_source - and a label string - "name_a".
+   ret = mv_face_recognition_register(handle, mv_source, "name_a");
+   if (ret != MEDIA_VISION_ERROR_NONE) {
+      // handle an error.
+   }
+
+   ret = mv_destroy_source(mv_source);
+   if (ret != MEDIA_VISION_ERROR_NONE) {
+       // handle an error.
+   }
+   ```
+
+   3.2 Unregister the face corresponding to a given label string, which is a process that face
+       recognition framework drops all data related to the given label string. In fact, this process is to retrain the internal model
+       without the dropped data.
+
+   ```
+   // Drop all data related to a given label string - "name_a".
+   ret = mv_face_recognition_unregister(handle, "name_a");
+   if (ret != MEDIA_VISION_ERROR_NONE) {
+       // handle an error.
+   }
+   ```
+
+   3.3 Recognize the given face image, which is a process that face recognition framework performs a forward propergation to find the most suitable label.
+
+   ```
+   // Recognize the given face image.
+   ret = mv_face_recognition_inference(handle, mv_source);
+   if (ret != MEDIA_VISION_ERROR_NONE) {
+       // handle an error.
+   }
+   ```
```

Comment:
```suggestion
    3.  Recognize the given face image, which is a process in which the face recognition framework performs a forward propagation to find the most suitable label.

        ```
        // Recognize the given face image.
        ret = mv_face_recognition_inference(handle, mv_source);
        if (ret != MEDIA_VISION_ERROR_NONE) {
            // handle an error.
        }
        ```
```

Reply:
Thanks.